docs/geedocs/5.3.3/answer/fusionAdministration/pushAndPublishDB.html

```diff
@@ -258,6 +258,7 @@ <h1>Push and publish databases<a class="headerlink" href="#push-and-publish-data
 </ol>
 <p class="rubric">Learn more</p>
 <ul class="simple">
+<li><a class="reference internal" href="../geeServerAdmin/publishDefaultDatabase.html"><span class="doc">Publish default database</span></a></li>
```

Comment:
Minor: on line 188 it may be a good idea to add "default database" to the list of things to specify. There are other things missing anyway, so I can go either way.